torch/quantization/quantize_fx.py

```diff
@@ -21,10 +21,10 @@ def fuse_fx(graph_module, inplace=False):
 
 def _prepare_fx(graph_module, qconfig_dict, inplace, is_dynamic_quant):
     _check_is_graph_module(graph_module)
-
+    graph_module = fuse_fx(graph_module, inplace)
     quantizer = Quantizer()
     prepare = quantizer.prepare_dynamic if is_dynamic_quant else quantizer.prepare
-    prepared = prepare(graph_module, qconfig_dict, inplace)
+    prepared = prepare(graph_module, qconfig_dict, inplace=True)
```

Comment:
should this keep the value of inplace?

Reply:
inplace is applied in fuse_fx step already, e.g. when inplace is False, the model is already copied, we don't need to copy again